src/main/java/com/thirdcc/webapp/repository/EventAttendeeRepository.java

```diff
@@ -21,4 +21,10 @@
 
     Optional<EventAttendee> findOneByEventIdAndUserId(Long eventId , Long userId);
 
+    @Query(value = "SELECT ea.id AS id, ea.user_id AS userId, ea.event_id AS eventId, ea.provide_transport AS provideTransport, u.first_name AS firstName, u.last_name AS lastName, ui.year_session AS yearSession "
+            + "FROM "
+            + "event_attendee ea JOIN jhi_user u ON u.id = ea.user_id "
+            + "JOIN user_uni_info ui ON u.id = ui.user_id "
+            + "WHERE ea.event_id = :eventId", nativeQuery = true)
+    Page<EventAttendeeCustomInterface> customFindAllByEventId(Pageable pageable, @Param("eventId") Long eventId);
```

Comment:
please avoid using a native query, cauz in the `test` env, we are using `h2` DB. There might be database incompatible issue.

Please use `JPQL` instead next time.
But since in the 9th April weekly meeting, we agreed to move the logic to the Java side, instead of using a query.

Reply:
Resolve by https://github.com/Five-Fishes/Club-Management-Spring-Server/pull/97/commits/4533e55ee4cb81ddd20ccd03b4a792ed650b472d